packages/api-client/src/user/UserAPI.ts

```diff
@@ -519,15 +527,18 @@ export class UserAPI {
    */
   public async postListUsers(
     users: {qualified_ids: QualifiedId[]} | {qualified_handles: QualifiedHandle[]},
-  ): Promise<User[]> {
+  ): Promise<UsersReponse> {
     const config: AxiosRequestConfig = {
       data: users,
       method: 'post',
       url: UserAPI.URL.LIST_USERS,
     };
 
-    const response = await this.client.sendJSON<User[]>(config);
-    return response.data;
+    const {data} = await this.client.sendJSON<User[] | UsersReponse>(config);
```

Comment:
can we rename data too like userData?

Reply:
This is the basic axios response, the `data` represents the raw payload. I can sure rename it if you feel it's going to be clearer